Makefile

```
@@ -257,6 +263,9 @@ ifdef DOCKER
 	@echo $(DOCKER_PASSWORD) | docker login --username "$(DOCKER_USERNAME)" --password-stdin $(DOCKER_REGISTRY)
 	@cd $(ROOT_DIRECTORY) && for DOCKER_FILE in $(shell ls -1 Dockerfile.*); do $(MAKE) docker-build DOCKER_FILE="$${DOCKER_FILE}"; done
 	@cd $(ROOT_DIRECTORY) && for DOCKER_FILE in $(shell ls -1 Dockerfile.*); do $(MAKE) docker-push DOCKER_FILE="$${DOCKER_FILE}"; done
+ifdef PCT_DOCKER_REPOSITORY
+	@echo $(PCT_DOCKER_PASSWORD) | docker login --username "$(PCT_DOCKER_USERNAME)" --password-stdin $(PCT_DOCKER_REGISTRY)
+endif
```

I don't get the logic here. 
The docker-push call may need to have the login done when PCT_DOCKER_REPOSITORY is set.
```suggestion
ifdef PCT_DOCKER_REPOSITORY
	@echo $(PCT_DOCKER_PASSWORD) | docker login --username "$(PCT_DOCKER_USERNAME)" --password-stdin $(PCT_DOCKER_REGISTRY)
endif
	@cd $(ROOT_DIRECTORY) && for DOCKER_FILE in $(shell ls -1 Dockerfile.*); do $(MAKE) docker-push DOCKER_FILE="$${DOCKER_FILE}"; done
```